Objects/typeobject.c

```diff
@@ -4663,6 +4663,8 @@ static PyMethodDef object_methods[] = {
     {0}
 };
 
+PyDoc_STRVAR(object_doc,
+"object()\n--\n\nThe superclass for all Python classes.");
```

Comment:
for -> of

Reply:
To me, 'of' has more of an implication of immediate superclass.  But see my response to Serhiy.